modules/ROOT/pages/sandboxes.adoc

```diff
@@ -0,0 +1,39 @@
+= Sandboxes
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+After you install an Anypoint API Community Manager update, it is not possible to undo the installation
+and return to a previous version. Follow the best-practice workflow by creating safe sandbox
+environments and using them to test each update before installing the update in your production environment.
+
+Best-practice workflow:
+
+. Keep a partial copy sandbox or a full copy sandbox as a long-term staging environment.
++
+The staging environment will have a partial or full copy of the data in the production environment.
+. Create a short-term developer sandbox.
+. Install the update in the developer sandbox and test it.
+. Discard the developer sandbox.
+. Install the update in the staging sandbox and test it.
+. Install the update in your production environment.
+
+To create a new sandbox:
+
+. In *Setup*, search for *Sandboxes* using the *Quick Find* box and click *New Sandbox*.
+. Enter a name and description and create a new developer sandbox.
```

Comment:
So...should the topic title include "Developer"?

Reply:
Does entering a name and description automatically start sandbox creation? There is no "enter" or "ok" action?